mmdet/core/evaluation/eval_hooks.py

```diff
@@ -10,16 +10,24 @@ class EvalHook(Hook):
     Attributes:
         dataloader (DataLoader): A PyTorch dataloader.
         interval (int): Evaluation interval (by epochs). Default: 1.
+        init_eval (bool): Evaluation before run. It is useful when loading a
+          checkpoint. Default: False
     """
 
-    def __init__(self, dataloader, interval=1, **eval_kwargs):
+    def __init__(self, dataloader, interval=1, init_eval=False, **eval_kwargs):
```

Comment:
We can add an argument `start`. When it is set to 0, we will perform evaluation before the first epoch. It also enables users to skip the first several epochs.